target/linux/pistachio/modules.mk

```diff
@@ -192,48 +192,3 @@ define KernelPackage/nhc_fragment/description
 endef
 
 $(eval $(call KernelPackage,nhc_udp))
```

Comment:
In stead of removing whole definition from modules.mk, how about just removing kmod-uccp420wlan and kmod-cfg80211 from target.mk. I think with that you won't need to add the patches of mac80211 too..